src/components/Launcher.js

```diff
@@ -11,7 +11,7 @@ class Launcher extends Component {
     super();
     this.state = {
       launcherIcon,
-      isOpen: false
+      isOpen: false,
```

Comment:
Let's be consistent with our commas across the repo. If there is not consistency already, this change is unwarranted